### ISSUE TITLE
ListItemCheckboxMixin causes ARIA attribute cannot be used: aria-labelledby

### ISSUE THREAD

**Issue by Vitalii-Misechko** [BrightspaceUI/core#1303]
Basically, axe-core 4.2.0 code states that 

>    // aria-label and aria-labelledy are not allowed on elements
>    // without roles, but if the element has text content then
>    // we will return undefined as a screen reader will read
>    // the text content

It seems that `_renderCheckboxAction` function renders a label that does not have an ARIA role according to `getRoles` of axe-core.

JFYI @bemailloux @johngwilkinson

**Comment by Vitalii-Misechko** [comment]
The issue can be reproduced using this test https://github.com/Brightspace/insights-engagement-dashboard/blob/0b276d36217ba7cd52f3e0e093125a73d3dd46b2/test/components/card-selection-list.test.js#L15
